sparkctl/cmd/create.go

```diff
@@ -267,28 +272,118 @@ func isContainerLocalFile(file string) (bool, error) {
 	return false, nil
 }
 
+type Blob interface {
```

Comment:
Looks like both `Blob` and `UploadHandler` are only used inside the `cmd` package, so they should both start with lower-case letters.